frontend/src/metabase/entities/containers/EntityForm.jsx

```diff
@@ -47,7 +48,10 @@ export default class EntityForm extends React.Component {
         form={form}
         initialValues={entityObject}
         onSubmit={onSubmit}
-        onSubmitSuccess={action => onSaved && onSaved(action.payload.object)}
+        onSubmitSuccess={action =>
+          action && onSaved && onSaved(action.payload.object)
+        }
```

Comment:
Did something change with `onSubmitSuccess` where action might not be passed now?